src/main/java/com/band/api/repository/User.java

```diff
@@ -0,0 +1,106 @@
+package com.band.api.repository;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name = "user", catalog = "band")
+public class User {
+
+    @Id
+    @GeneratedValue(strategy= GenerationType.IDENTITY)
+    private Integer id;
+
+    @Column(columnDefinition = "VARCHAR(128)", nullable = false)
+    private String username;
+
+    @Column(columnDefinition = "VARCHAR(128)", nullable = false)
+    private String email;
+
+    @Column(columnDefinition = "CHAR(60)", nullable = false)
+    private String password_hash;
+
+    @Column(columnDefinition = "CHAR(60)")
+    private String recovery_hash;
+
+    @Column(columnDefinition = "VARCHAR(128)")
+    private String recovery_question;
+
+    @Column(columnDefinition = "VARCHAR(128)", nullable = false)
+    private String name;
+
+    @Column(columnDefinition = "VARCHAR(128)", nullable = false)
+    private String state;
+
+    @Column(columnDefinition = "VARCHAR(128)", nullable = false)
+    private String city;
+
+
+
+    public User() {
+    }
+
+    public String getUsername() {
```

Comment:
Remove all of the getters & setters -- read about lombok https://projectlombok.org/features/GetterSetter

Can use @Getter @Setter annotation at the field level or classes level, and can use @Data to apply both getter and setter.